runhouse/servers/http/http_server.py

```diff
@@ -119,6 +120,9 @@ async def ainitialize(
         *args,
         **kwargs,
     ):
+        from runhouse.logger import logger
+
+        logger.setLevel(kwargs.get("logs_level", DEFAULT_LOG_LEVEL))
```

Comment:
we re-use this further down, prob worth setting to a variable (`kwargs.get("logs_level", DEFAULT_LOG_LEVEL)`)